### PR TITLE
ndndpdk-depends.sh: fix quoting

### PR DESCRIPTION
**Pull request by Pesa**
Untested, but the changes should be pretty straightforward.

Draft because... see comment inline.

### CODE REVIEW

File: docs/ndndpdk-depends.sh [hunk 1]
```diff
@@ -282,66 +282,66 @@ if [[ $CONFIRM -ne 1 ]]; then
...
 
-if [[ $NODEVER != '0' ]]; then
-  curl -sfL ${NDNDPDK_DL_NODESOURCE_DEB}/setup_${NODEVER} | $SUDO bash -
+if [[ $NODEVER != 0 ]]; then
+  curl -sfL "${NDNDPDK_DL_NODESOURCE_DEB}/setup_${NODEVER}" | $SUDO bash -
   APT_PKGS+=(nodejs)
 fi
 
 APT_PKG_LIST="${APT_PKGS[@]}"
 $SUDO sh -c "DEBIAN_FRONTEND=noninteractive apt-get -y -qq install ${APT_PKG_LIST}"
```
Comment (Pesa) [inline docs/ndndpdk-depends.sh:307]: I wanted to get rid of the `APT_PKG_LIST` intermediate variable, but then I noticed this strange line... why are you using `sh -c` to run apt-get?

Comment (yoursunny) [reply docs/ndndpdk-depends.sh:307]: This is learned from https://get.docker.com script.

Comment (Pesa) [reply docs/ndndpdk-depends.sh:307]: but why?

Comment (yoursunny) [reply docs/ndndpdk-depends.sh:307]: No particular reason from me.
I simply followed existing examples.

**Comment by Pesa** [comment]
CI complains about the Dockerfile

```
Line 7: RUN apt-get -y -qq update && \
Issue  Category      Title                 Description
    1  Optimization  apt-get update with   Use of apt-get update should be paired with rm -rf
                     matching cache rm     /var/lib/apt/lists/* in the same layer.
```

seems to be a preexisting issue.

**Review by yoursunny — COMMENTED** [review]
Will merge after testing.
Don't worry about unit test failure in CI.

**Pull request merged**